app/src/main/java/com/sopt/now/compose/BottomBar.kt

```diff
@@ -0,0 +1,85 @@
+package com.sopt.now.compose
+
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Home
+import androidx.compose.material.icons.filled.Person
+import androidx.compose.material.icons.filled.Search
+import androidx.compose.material3.Icon
+import androidx.compose.material3.NavigationBar
+import androidx.compose.material3.NavigationBarItem
+import androidx.compose.material3.NavigationBarItemDefaults
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.navigation.NavHostController
+import androidx.navigation.compose.currentBackStackEntryAsState
+
+
+@Composable
+fun BottomBar(
+    navController: NavHostController,
+    modifier: Modifier = Modifier,
+    isLoggedIn: Boolean
+) {
```

Comment:
Kotlin에서는 모든 선택적 패러미터(Optional Parameter, 디폴트 값이 주어진 패러미터)는 필수 패러미터 뒤에 배치되어야합니다. 그리고 컴포즈에서는 모든 선택적 패러미터 제일 위에 Modifier가 배치되어야 합니다...라고 코딩 컨벤션에는 적혀 있습니다.

필수는 아니지만 공통 룰을 지켜보면서 코드 가독성 올리는 것에 도움이 될 수 있을 것 같아서 공유드립니다.

```suggestion
fun BottomBar(
    navController: NavHostController,
    isLoggedIn: Boolean,
    modifier: Modifier = Modifier,
) {
```